src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form"
+      data-qa="big"
+    >
```

Comment:
In HTML5, the `<form>` tag cannot be self-closing. You should use a separate closing tag `</form>` instead of a self-closing tag.